build.sh does not work

**Describe the bug**
<!-- A clear and concise description of what the bug is. -->

**To Reproduce**
Steps to reproduce the behavior:
1. git clone the tfsec repo
2. c tfsec
3. bash scripts/install.sh
4. See error

**Expected behavior**
I expect to be able to build tfsed.

**Output of your tfsec command with --debug flag**

cmd/tfsec/main.go:8:2: cannot find package "github.com/aquasecurity/tfsec/internal/app/tfsec/cmd" in any of:
	/usr/src/github.com/aquasecurity/tfsec/internal/app/tfsec/cmd (from $GOROOT)
	/home/nkadel/go/src/github.com/aquasecurity/tfsec/internal/app/tfsec/cmd (from $GOPATH)
cmd/tfsec-checkgen/main.go:16:2: cannot find package "github.com/AlecAivazis/survey/v2" in any of:
	/usr/src/github.com/AlecAivazis/survey/v2 (from $GOROOT)
	/home/nkadel/go/src/github.com/AlecAivazis/survey/v2 (from $GOPATH)


This is on RHEL 8 with /usr/bin/go and a previous successful of the tfsec binaries with all the dependencies.

`
 - OS: `?`

**Example Code**
bash scripts/install.sh

**Additional context**

Building it locally would allow some potential debugging, a security assessment of the code itself, and potentially RPM packaging rather than directly relying on your published binary. I'd be happy to help you do that for RHEL users like myself.

--- a/Makefile
+++ b/Makefile
@@ -16,6 +16,7 @@ test:
 .PHONY: build
 build:
 	./scripts/build.sh
+	
 
 .PHONY: generate-docs
 generate-docs:

--- a/scripts/build.sh
+++ b/scripts/build.sh
@@ -1,15 +1,15 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
 BINARY=tfsec
 CHECK_GEN_BINARY=tfsec-checkgen
 TAG=${TRAVIS_TAG:-development}
 GO111MODULE=on
 export CGO_ENABLED=0
-export GOFLAGS=-mod=vendor
 args=(-ldflags "-X github.com/aquasecurity/tfsec/version.Version=${TAG} -s -w -extldflags '-fno-PIC -static'")
 
 mkdir -p bin/darwin
 GOOS=darwin GOARCH=amd64 go build -o bin/darwin/${BINARY}-darwin-amd64 "${args[@]}" ./cmd/tfsec/
-GOOS=darwin GOARCH=amd64 go build -o bin/darwin/${CHECK_GEN_BINARY}-darwin-amd64 "${args[@]}" ./cmd/tfsec-checkgen/
+GOOS=darwin GOARCH=amd64 go build -o ./bin/darwin/${CHECK_GEN_BINARY}-darwin-amd64 "${args[@]}" ./cmd/tfsec-checkgen/
 mkdir -p bin/linux
 GOOS=linux GOARCH=amd64 go build -o bin/linux/${BINARY}-linux-amd64 "${args[@]}" ./cmd/tfsec/
 GOOS=linux GOARCH=amd64 go build -o bin/linux/${CHECK_GEN_BINARY}-linux-amd64 "${args[@]}" ./cmd/tfsec-checkgen/

